.github/pull_request_template.md

```diff
@@ -1,10 +1,17 @@
 Jira: [CP-XXX]
 
-**Description**
+# Description
+
+## Checklist before requesting a review - nice to have
 
 - [ ] getState function in async thunk actions is correctly typed
 - [ ] redux selectors are used in components / prop drilling is reduce
 
+## Checklist before merging a pull request
+
```

Comment:
Not sure if we should add it here :)

Reply:
I wanted us to have a simple checklist to remind ourselves of just before merging ☺️  Where could such a checklist be placed so it doesn't become a forgotten document? 🤔